docs/markdown/Machine-files.md

```diff
@@ -84,21 +121,76 @@ command line will override any options in the native file. For example, passing
 
```

Comment:
Drive-by:

```diff
- *New in native files in 0.54.0*, always in cross files.
+ *available in native files since 0.54.0*
+ *always available in cross files*
```